openmdao/docs/openmdao_book/features/core_features/working_with_components/distributed_components.ipynb

```diff
@@ -37,45 +37,21 @@
    "cell_type": "markdown",
```

Comment:
<p>suggest: "... usually creates one automatically as an output of a component referred to as an 'Auto-IVC'."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO/pull/2218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>reworded</p>